lib/rucio/core/rule.py

```diff
@@ -1180,14 +1180,18 @@ def update_rule(rule_id, options, session=None):
     :raises:            RuleNotFound if no Rule can be found, InputValidationError if invalid option is used, ScratchDiskLifetimeConflict if wrong ScratchDiskLifetime is used.
     """
 
-    valid_options = ['comment', 'locked', 'lifetime', 'account', 'state', 'activity', 'source_replica_expression', 'cancel_requests', 'priority', 'child_rule_id', 'eol_at', 'meta', 'purge_replicas', 'boost_rule']
+    valid_options = ['comment', 'locked', 'lifetime', 'account', 'state',
+                     'activity', 'source_replica_expression', 'cancel_requests',
+                     'priority', 'child_rule_id', 'eol_at', 'meta',
+                     'purge_replicas', 'boost_rule']
```

Comment:
Isn’t this purely a cosmetic change? I find that bundling it in a commit with functional changes can be confusing.

Reply:
it is a purely cosmetic change, i can roll it back if you feel strongly about that.

Reply:
No, I do not object, but I would advise having a separate commit for this. Same for the addition of type hints (especially in unrelated code).